akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -53,6 +53,30 @@ final class Flow[-In, +Out, +Mat](private[stream] override val module: Module)
   def via[T, Mat2](flow: Graph[FlowShape[Out, T], Mat2]): Flow[In, T, Mat] = viaMat(flow)(Keep.left)
 
   /**
+   * Transform this [[Flow]] by appending a Reactive Streams compliant [[org.reactivestreams.Processor]]. This
+   * method takes a factory that instantiates the [[org.reactivestreams.Processor]] for every materialization
+   * of the resulting [[Flow]].
+   *
+   * {{{
+   *     +---------------------------------+
+   *     | Resulting Flow                  |
+   *     |                                 |
+   *     |  +------+        +-----------+  |
+   *     |  |      |        |           |  |
+   * In ~~> | this | ~Out~> | processor | ~~> T
+   *     |  |      |        |           |  |
+   *     |  +------+        +-----------+  |
+   *     +---------------------------------+
+   * }}}
+   * The materialized value of the combined [[Flow]] will be the materialized
+   * value of the current flow.
+   */
+  def viaProcessor[T](processorFactory: () ⇒ Processor[Out @uncheckedVariance, T]): Flow[In, T, Mat] = {
```

Comment:
> No, that's -your- logic.

No, you said:

> transform is not called viaTransform

Which I corrected, because it would be viaStage by that logic where something looks like viaX(X). 

> My logic is to drop -via- for everything but piecing together sources with flows or flows with flows.

Well, we cannot drop "via" for Flows. We can introduce a factory from Processor to Flow, that works fine. I don't like it because it is extra boilerplate though to write. 

> According to my logic, transform ought to be called stage.

Why rename after the final RC? If "viaProcessor" offends your eyes that much, then I introduce a Flow.apply() that takes a processor.